File: Scala/abc199_A/Main.scala
@main def Main(): Unit =
  val lines = scala.io.StdIn.readLine()
  val result = solve(lines)
  println(result)

def solve(input: String): String =
  val Array(a, b, c) = input.split(" ").map(_.toInt)
  if (a * a + b * b < c * c) "Yes" else "No"

File: Dart/abc199_a/main.dart
import 'dart:io';

void main() {
  final line = stdin.readLineSync()!;
  final parts = line.split(" ").map(int.parse).toList();

  final a = parts[0];
  final b = parts[1];
  final c = parts[2];

  final result = solve(a, b, c);
  print(result);
}

String solve(int a, int b, int c) => (a * a + b * b < c * c) ? "Yes" : "No";
